spec/controllers/synapses_controller_spec.rb

```diff
@@ -35,13 +35,30 @@
       end
     end
 
+    context 'with private topics' do
+      it 'redirects to /' do
```

Comment:
I updated that not long ago and thought I made an improvement, but only for some cases :p